salt/utils/parsers.py

```diff
@@ -911,9 +911,17 @@ def check_running(self):
         Check if a pid file exists and if it is associated with
         a running process.
         '''
+        # There is no os.getppid method for windows
+        from salt.utils import is_windows
+        if is_windows():
+            from salt.utils.win_functions import get_parent_pid
```

Comment:
This file already has 'import salt.utils' at the top. So I would skip the import and just use:

```
if salt.utils.is_windows():
```